python/tutorials/03-matrix-multiplication.py

```diff
@@ -274,6 +283,19 @@ def matmul_kernel(
     pid_m = first_pid_m + ((pid % num_pid_in_group) % group_size_m)
     pid_n = (pid % num_pid_in_group) // group_size_m
 
+    # -----------------------------------------------------------
+    # Add some integer bound assumptions. This helps to guide 
```

Comment:
Nit: Start "This helps.." from the next line to follow existing comment style. :)